dev.py

```diff
@@ -416,6 +416,8 @@ class Build(Task):
               "build and build-install directories)."))
     debug = Option(
         ['--debug', '-d'], default=False, is_flag=True, help="Debug build")
+    release = Option(
+        ['--release', '-d'], default=False, is_flag=True, help="Release build")
```

Comment:
Probably copy-paste thing, `python dev.py build -d` will produce a `release` build. Do we want `-r` or something here rather than reusing `-d`?